src/components/ui/Icon/README.md

```diff
@@ -0,0 +1,167 @@
+
+You can customise the margin value to ensure proper placement in your layout. To maintain visual consistency, Nautilus uses the spacing values provided by your theme by default.
+
+Aligning an icon perfectly to text it's inline with can sometimes be tricky. By default, the container is set to vertical-align middle, but you can override this in your CSS if needed.
+
+### Border & background
+
+An icon can have a border and a background. These use a circular shape by default because it looks nicest, but this value, along with border width, can be changed in your theme.
+
+```jsx
+<Icon name="mic" background="#181b1c" color="#cd2f83" />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" />
+<Icon name="mic" background="#cd2f83" border="#181b1c" color="#181b1c" />
+<Icon name="mic" background="#181b1c" color="#fff" />
+<br />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" extraSmall />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" small />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" medium />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" large />
+<Icon name="mic" border="#cd2f83" color="#cd2f83" extraLarge />
+```
+
+This is generally best when an `Icon` is used inside a `Tab` or a `Button`, since it gives the icon additional emphasis and creates a clear touch target.
+
+## Accessibility
+
+### An icon + text is the best choice for accessibility
+
+Icons can help people who have trouble processing text or who are trying to navigate a product written in a foreign language. Use icons to increase the "scan-ability" of large blocks of content.
+
+Remember that an icon may have different meanings in different contexts, or when interpreted through the lenses of different cultures, and they provide no meaning to people with vision loss.
+
+While an icon can be used on its own in constrained UIs, the best practise is to **pair an icon with a visual text label**, especially for important actions that could possibly be misunderstood. There are some icons that are universally understood (a trash icon for deleting, an "X" icon for closing a window), but there is often room for misinterpretation.
+
+### Don't overuse icons
+
+An interface littered with icons can lead to cognitive overload. Use icons sparingly to highlight important actions and information, not just for decoration.
+
+### Keep internationalisation in mind
+
+When using an icon to represent text, be mindful of how that icon might be interpreted in other countries. For instance, a dollar sign (`$`) icon represents money in many countries, but not all. In these cases, localise your icons for your visitors or use a more generic, universally recognised icon.
+
+### Use a title for icons without a text label
+
+If an icon is used without a text label, it needs to have a text alternative so that screen readers can read it. This adds a `title` tag inside the `svg` that’s conveyed to screen reader users.
+
+Best practises for label text:
```

Comment:
Should this have a newline after it? 🤷‍♂

Reply:
Not really, since it's a lead-in paragraph to the list and warrants being in closer proximity to the list that it's a part of. 

Current styling does add its own newline though:
<img width="640" alt="Screenshot 2019-07-01 at 22 54 10" src="https://user-images.githubusercontent.com/376315/60468994-3bbd6f80-9c53-11e9-9684-5ac73b2b5e3e.png">